.env.example

```diff
@@ -0,0 +1,4 @@
+API_ENDPOINT=https://api-staging.travis-ci.org
```

Comment:
By default locally we use production API, so I'd suggest to change this:
```suggestion
# API_ENDPOINT=https://api.travis-ci.org
```
Also I think we should comment it out by default, because it gets computed depending on `TRAVIS_PRO` value.